storage/sql/transactions/get.sql

```diff
@@ -13,32 +13,32 @@
  */
 
 SELECT
-	"t_id" as "id",
-	"b_id" as "blockId",
-	"b_height" as "height",
-	"t_type" as "type",
-	"t_timestamp" as "timestamp",
-	"t_senderId" as "senderId",
-	"t_recipientId" as "recipientId",
-	"t_amount" as "amount",
-	"t_fee" as "fee",
-	"t_signature" as "signature",
-	"t_signSignature" as "signSignature",
-	regexp_split_to_array("t_signatures", ',') as "signatures",
-	"t_senderPublicKey" as "senderPublicKey",
-	"t_recipientPublicKey" as "recipientPublicKey",
-	"t_requesterPublicKey" as "requesterPublicKey",
-	(( SELECT (blocks.height + 1)
-           FROM blocks
-          ORDER BY blocks.height DESC
-         LIMIT 1) - b.height) AS confirmations
-FROM
-	(full_blocks_list fbl
-	LEFT JOIN blocks b ON (((fbl."b_id")::text = (b.id)::text)))
+	trs."id" AS "id",
+	b."height" AS "height",
+	trs."blockId" AS "blockId",
+	trs."type" AS "type",
+	trs."timestamp" AS "timestamp",
+	encode(trs."senderPublicKey", 'hex'::text) AS "senderPublicKey",
+	encode(m."publicKey", 'hex'::text) AS "recipientPublicKey",
+	upper(trs."senderId"::text) AS "senderId",
+	upper(trs."recipientId"::text) AS "recipientId",
```

Comment:
We were not using `upper` int he view before, why you are introducing it?

Reply:
We were using it in `trs_list` view. Btw, it looks like we can get rid of `trs_list` view as well. What do you think?

Reply:
Yes if not in use drop that view.